### PR TITLE
Add Event to schema.

### PR DESCRIPTION
**Pull request by liuche**
This is the schema for the events part of the Firebase, which can be joined via the id.

### CODE REVIEW

**Comment by liuche** [comment]
@fluffyemily let me know if you have thoughts on this! Basically, I decided that the top level of filtering would be whether the venue existed as a card (e.g., meets the criteria for being shown). After that, the start/endTimes should provide enough information for filtering/ordering by time, right?

File: schema.md [hunk 1]
```diff
@@ -56,3 +57,20 @@ Place: {
...
+Event: { 
+    coordinates: {
+        lat: #,
+        lng: #
+    },
+    description: "", # Used for badging
+    notification: "",
+    startTime: "", # RFC 3339, e.g., "2016-11-10T18:00:00-10:00",
+    endTime: "", # RFC 3339
+    id: "<yelp-id>" # Best effort Yelp id
```
Comment (mcomella) [inline schema.md:73]: nit: missing comma (yay 👋 JSON! 👋)

File: schema.md [hunk 2]
```diff
@@ -56,3 +57,20 @@ Place: {
...
+
+## Event
+
+```
+Event: { 
+    coordinates: {
+        lat: #,
+        lng: #
+    },
+    description: "", # Used for badging
```
Comment (mcomella) [inline schema.md:69]: The UX mocks (via https://github.com/mozilla-mobile/prox/issues/99) show text, followed by an underlined text acting as a CTA to click for more info. I think we should separate description into an array (or two keys), where the first part is the normal text and the second part is the underlined text.

**Review by mcomella — APPROVED** [review]
lgtm w/ changes.

File: schema.md [hunk 3]
```diff
@@ -56,3 +57,20 @@ Place: {
...
+    notification: "",
+    startTime: "", # RFC 3339, e.g., "2016-11-10T18:00:00-10:00",
+    endTime: "", # RFC 3339
+    id: "<yelp-id>" # Best effort Yelp id
+    url: ""
```
Comment (mcomella) [inline schema.md:74]: To be clear, this url is the website we should open when the event is clicked, right?

File: schema.md (hunk 1 above)
Comment (liuche) [inline schema.md]: Yep!

File: schema.md (hunk 2 above)
Comment (liuche) [reply schema.md:69]: I was assuming that "More info" would be used consistently across all badging.

File: schema.md (hunk 3 above)
Comment (liuche) [reply schema.md:74]: Yep!

**Comment by liuche** [comment]
[As a user I want my app to be in sync with the events stored in the firebase](https://trello.com/c/5PYxdd1V/212-as-a-user-i-want-my-app-to-be-in-sync-with-the-events-stored-in-the-firebase)

**Pull request merged**